tck/src/main/java/org/eclipse/microprofile/lra/tck/service/LRATestService.java

```diff
@@ -0,0 +1,88 @@
+/*
+ *******************************************************************************
+ * Copyright (c) 2019 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package org.eclipse.microprofile.lra.tck.service;
+
+import org.eclipse.microprofile.lra.tck.LRAClientOps;
+import org.eclipse.microprofile.lra.tck.LraTckConfigBean;
+import org.eclipse.microprofile.lra.tck.participant.api.Util;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.ClientBuilder;
+import javax.ws.rs.client.WebTarget;
+import java.net.MalformedURLException;
+import java.net.URI;
+import java.net.URL;
+
+@ApplicationScoped
+public class LRATestService {
+
+    @Inject
+    LRAMetricService lraMetricService;
+    
+    @Inject
+    LraTckConfigBean config;
+
+    private LRAClientOps lraClient;
+
+    private static Client tckSuiteClient;
+
+    private WebTarget tckSuiteTarget;
+
+    public void prepareState() {
+        tckSuiteClient = ClientBuilder.newClient();
+        lraMetricService.clear();
+
+        try {
+            tckSuiteTarget = tckSuiteClient.target(URI.create(new URL(config.tckSuiteBaseUrl()).toExternalForm()));
+            lraClient = new LRAClientOps(tckSuiteTarget);
+        } catch (MalformedURLException mfe) {
+            throw new IllegalStateException("Cannot create URL for the LRA TCK suite base url " + config.tckSuiteBaseUrl(), mfe);
+        }
+    }
+
+    public void clearState() {
+        if(tckSuiteClient != null) {
+            tckSuiteClient.close();
+        }
+    }
+
+    public LRAClientOps getLraClient() {
+        return lraClient;
+    }
+
+    public WebTarget getTckSuiteTarget() {
+        return tckSuiteTarget;
+    }
+
+    public String lraClientId(Class<?> clazz, String addition) {
```

Comment:
The other getters are prefixed with get so this one should be too (ie getLRAClientId). BTW I made an earlier comment about the addition of this method (it is only ever used once).